web/client/plugins/StreetView/StreetView.jsx

```diff
@@ -40,9 +40,11 @@ const StreetViewPluginContainer = connect(() => ({}), {
  * @memberof plugins
  * @property {string} cfg.apiKey The API key to use.
  * Mandatory in production. In order to allow fine billing strategies (with different API keys), the API key can be defined and customized here in this configuration option or in `localConfig.json` at, in order of priority one of:
```

Comment:
* Mandatory in production. In order to allow fine billing strategies (with different API keys), the API key can be defined and customized here in this configuration option or in `localConfig.json` with the following order of priority:

Reply:
```suggestion
 * Mandatory in production. In order to allow fine billing strategies (with different API keys), the API key can be defined and customized here in this configuration option or in `localConfig.json` with the following order of priority:
```